app/templates/index.html

```diff
@@ -41,5 +41,38 @@
         </div>
         <h1>{{ title }}</h1>
     </div>
+
```

Comment:
Maybe it would be nice to wrap the whole section (including its title) in a div with a class (for example) `class="work-section"` and then, for the class that surrounds the content of each work experience, you could rename it (for example) `class="work-section-content"`, in this way it is clearer to know where each part of the section begins and ends.